frontend/src/app/profile/profile-editor/profile-editor.component.ts

```diff
@@ -78,6 +81,14 @@ export class ProfileEditorComponent implements OnInit {
   onSubmit(): void {
     if (this.profileForm.valid) {
       Object.assign(this.profile, this.profileForm.value);
+      if (this.profile.accepted_community_agreement === undefined) {
```

Comment:
Going to change this to `!this.profile.accepted_community_agreement` which handles the case of undefined but also handles the case of `false`, which is actually more expected based on the type of `profile` being `Profile`.